src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html

```diff
@@ -24,7 +24,7 @@
         </div>
         <div class="col-lg-3 col-md-6 col-sm-6 align-self-center" *ngIf="isExamMode && !isTestRun">
             <ng-container *jhiHasAnyAuthority="['ROLE_ADMIN', 'ROLE_INSTRUCTOR']">
-                <ng-container *ngIf="exam?.numberOfCorrectionRoundsInExam > 1">
+                <ng-container *ngIf="isAtLeastInstructor && exam?.numberOfCorrectionRoundsInExam > 1">
```

Comment:
couldn't you move the *ngIf into the parent <div>? So in case it is false the whole div-component will not be rendered at all?

Reply:
done, good catch!